plugins/ConditionalRestrictions.py

```diff
@@ -88,16 +88,22 @@ def way(self, data, tags, nds):
           condition_started = True
         elif c == "(":
           parentheses += 1
+          if not condition_started:
+            err.append({"class": 33501, "subclass": 0 + stablehash64(tag + '|' + tag_value), "text": T_("Missing `@` in \"{0}\"", tag)})
+            bad_tag = True
+            break
         elif c == ")":
           parentheses -= 1
           if parentheses == -1:
             err.append({"class": 33501, "subclass": 2 + stablehash64(tag + '|' + tag_value), "text": T_("Mismatch in the number of parentheses in \"{0}\"", tag)})
             bad_tag = True
             break
         elif c == ";" and parentheses == 0:
+          if not condition_started:
```

Comment:
Note to self: move to elif line; that way tmp_str remains intact if we ever want to use it